src/index.js

```diff
@@ -1,6 +1,16 @@
-const fibonacci = (n) => {
+// Generators are a special class of functions that simplify the task of writing iterators.
+function* fib(n, current = 1, next = 1) {
+ // if (n === 0) {
+   // return 0;
+  // }
```

Comment:
try to keep your code clean. If you don't want to lose a part of your code, better commit.